notes.md

```diff
@@ -27,3 +27,19 @@ POST: not idempotent (some effect)
 - Add new endpoint? Get all drafts for user?
 - Continue to save drafts every 2 seconds? Or use separate button? (UX)
 - Make PR
+
+# Design
+
+- Where and how do we store data (drafts)?
+    - Reuse existing table `user_drafts`
+- How to store different drafts (multiple drafts per user)?
+    - Change the API endpoint to accept dynamic `draftName`
+- How to get all the drafts?
+    - Add a new API endpoint GET to read all the user-related drafts: GET `api/users/<user>/drafts
+
+- How will the feature work (behavior)?
```

Comment:
would be great to have some more details here. E.g. how do we edit the name? do we delete the draft on that separate page or on the editor? This currently looks like a list of "things to do", but it would be great to see more "UX decisions" instead.